<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckWindow</class>
 <widget class="QMainWindow" name="CheckWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Compliance Check Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>584</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Compliance Check IFC against IDS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_ifc_title">
         <property name="text">
          <string>IFC file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_ifc">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>580</width>
           <height>19</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>580</width>
           <height>19</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lbl_ids_file">
         <property name="text">
          <string>IDS file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_ids">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>580</width>
           <height>19</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>580</width>
           <height>19</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="lbl_ids_file_2">
         <property name="text">
          <string>Report type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_report">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>580</width>
           <height>19</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>580</width>
           <height>19</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>BCF</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HTML</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ODS</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="btn_check">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Check</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_back">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbl_notification">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>711</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
